app/ui/tooloptions.ui

```diff
@@ -187,6 +187,16 @@
          </property>
         </widget>
        </item>
+       <item>
+        <widget class="QCheckBox" name="useClosedBox">
+         <property name="toolTip">
+          <string>Close Polyline path (hold Ctrl to temporarily invert)</string>
```

Comment:
Although I now that we do this elsewhere in the file, referring to ctrl is actually not correct, because Control is substituted by ⌘ or Cmd on macOS.

I considered changing this in your PR but since we do it in various places, we'll keep it as is.

The correct way to fix this though would be something like:
```
QString controlShortcutText = QKeySequence(Qt::Key_Control).toString(QKeySequence::NativeText);
ui->useClosedBox->setToolTip(tr("Close Polyline path (hold %1 to temporarily invert)").arg(shortcutText));
```